packages/cli/src/commands/node/start.ts

```diff
@@ -21,8 +21,7 @@ export class StartNode extends Command {
     const config = await getSwankyConfig();
     // run persistent mode by default. non-persistent mode in case flag is provided.
     await execa.command(`${config.node.localPath} \
-      --rpc-cors http://localhost:*,http://127.0.0.1:*,https://localhost:*,https://127.0.0.1:*,https://polkadot.js.org,https://contracts-ui.substrate.io/ \
-      ${flags.tmp ? "--dev" : ""}`, {
+      ${flags.tmp ? "--dev" : "--rpc-cors http://localhost:*,http://127.0.0.1:*,https://localhost:*,https://127.0.0.1:*,https://polkadot.js.org,https://contracts-ui.substrate.io/"}`, {
```

Comment:
Maybe adding the CORS string to a variable above would make it a bit more readable?